### ISSUE TITLE
get authors to update gravatar

### ISSUE THREAD

**Issue by tristandenyer** [postmanlabs/postman-blog#210]
Need to write some instructions on how they can do that.

For launch, we could just show a postmanaut icon as a fallback image.

**Comment by ch264** [comment]
@michaelbettendorf is this ticket for you? Not sure who is relaying tickets for Wordpress users

**Comment by michaelbettendorf** [comment]
Yes, @ch264, I believe this ticket should ultimately fall to me. I'll have to do some research on how to update gravatars, and then I can write some instructions for authors.

**Comment by michaelbettendorf** [comment]
@Linda-Koffman

**Comment by ch264** [comment]
There is a bug on Wordpress with their Gravatar integration. I tried to updated my user profile picture yesterday and it updates on the notifications but not in my profile. I am looking into this

**Comment by ch264** [comment]
This is what I found in the Gravatar support pages: 
https://en.gravatar.com/support/contact-us/

I changed my gravatar, but I still see my old one on sites. What happened?
We cache your gravatars for periods of time, which means that your new gravatar will take effect after the old one expires from the cache. The reason we do this is to be able to serve your (and indeed everybody's) gravatar images much more quickly and reliably.

**Comment by ch264** [comment]
**Solution to updating Avatar for employees with @getpostman email address**

The Gravatar account linked to Wordpress is still @getpostman, but the Wordpress login has changed to @postman.

Please create a new account with your @postman email in Gravatar and uploaded your image in the new account, which will update your Wordpress user profile image
